src/main/resources/beam-template.conf

```diff
@@ -289,6 +289,20 @@ beam.beamskimmer.writeAllModeSkimsForPeakNonPeakPeriodsInterval = "int | 0"
 beam.beamskimmer.writeObservedSkimsPlusInterval = "int | 0" # Generic skimmer with a "label" and a "value" of type Double
 beam.beamskimmer.writeFullSkimsInterval = "int | 0"
 
+#SkimManager
+beam.skimmanager.h3.resolution = "int | 10"
+beam.skimmanager.h3.lowerBoundResolution = "int | 10"
+beam.skimmanager.keepTheNLastestSkims = "int | 1"
+beam.skimmanager.aggregateFunction = "String | LATEST_SKIM"
+#SkimManager - skimmer
```

Comment:
what is difference between "skimmer" and "odskimmer"? is the first a 1d skimmer? if so let's say that in the name to make it more clear.... most people in travel demand modeling associate the word "skimmer" as an OD matrix, so if we are not doing that we should make it explicit in the naming scheme.

Reply:
After some thoughts. I think this class should be called CountSkimmer with an integer instead of a double that would have as main function to count occurrences (like how many persons boarded a rh vehicle in certain space & time). Also because if we mix different kind of information in the same skimmer, we would have to design different aggregation methods for each kind of information. We already have this mechanism by implementing a separate skimmer.

Reply:
After some thoughts. I think this class should be called CountSkimmer with an integer instead of a double that would have as main function to count occurrences (like how many persons boarded a rh vehicle in certain space & time). Also because if we mix different kind of information in the same skimmer, we would have to design different aggregation methods for each kind of information. We already have this mechanism by implementing a separate skimmer.